fnl/paperplanes/providers/srht.fnl

```diff
@@ -0,0 +1,31 @@
+(local {: set-field : opts-to-fields} (require :paperplanes.util.providers))
+(local fmt string.format)
+
+(fn make [content-arg meta opts]
```

Comment:
Probably add an assert on opts.token so provide a more accurate "you must set provider_opts.token to your sr.ht token" or something?

Reply:
The error message is clear enough in that case
```
Error executing vim.schedule lua callback: vim/_editor.lua:0: paperplanes got no url back from provider: {"errors": [{"reason": "Invalid or expired OAuth token"}]}                                                 
stack traceback:                                                                                                                                                                                                    
        [C]: in function ''                                                                                                                                                                                         
        vim/_editor.lua: in function <vim/_editor.lua:0>  
```

Reply:
done